src/main/java/io/sentry/telemetry/SentryTelemetryService.java

```diff
@@ -160,7 +160,8 @@ private static class InfoOutput {
       final @Nullable String infoOutput = cliRunner.runSentryCli(String.join(" ", command), false);
       if (infoOutput != null) {
         final InfoOutput info = new InfoOutput();
-        info.isSaas = infoOutput.matches("(?m)Sentry Server: .*sentry.io$");
+        Pattern serverPattern = Pattern.compile("Sentry Server: .*sentry.io$", Pattern.MULTILINE);
+        info.isSaas = serverPattern.matcher(infoOutput).find();
 
         final @NotNull Pattern orgRegex =
             Pattern.compile("(?m)Default Organization: (.*)$", Pattern.MULTILINE);
```

Comment:
Is there a particular reason the `(?m)` part has been removed in the `serverPattern` but not the `orgRegex`?

Reply:
`(?m)` and `Pattern.MULTILINE` are the same thing, so `orgRegex` has it twice.

Reply:
Removed and tested without `(?m)`. Works.